docs/write_to_firestore.md

```diff
@@ -0,0 +1,36 @@
+# Example 6: Send model data to Google Firestore
+Starting version 0.1.30, fal lets you easily send data to [Google Firestore](https://cloud.google.com/firestore/). This example assumes that Firestore is [already enabled](https://cloud.google.com/firestore/docs/quickstart-servers#create_a_in_native_mode_database) in your Google Cloud project.
+
+## Credentials
+By default fal will attempt to use the credentials provided in profiles.yml. If that's unsuccessful, credentials environment variable is checked. See [here](https://cloud.google.com/docs/authentication/getting-started) for more information on how to set such environment variable.
+
+## Meta tag
+In a `schema.yml` file, within a target model, a meta tag should be added in order to connect the model to fal:
+```yaml
+meta:
+  fal:
+    scripts:
+      - path_to_fal_script.py
+```
+
+## `write_to_firestore` magic function
+This function is available in fal runtime. There is no need to import it. Here's an example invocation:
+
+```python
+write_to_firestore(df=dataframe, collection=collection_name, key_column=key_column)
```

Comment:
You know we can make naming the parameters mandatory by changing `write_to_firestore` function definition. Do we want that?

Reply:
It's ok to give users flexibility here.